src/SharpBrick.PoweredUp.BlueGigaBLE/BlueGigaBLEPoweredUpBluetoothCharacteristic.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Bluegiga.BLE.Events.ATTClient;
+using Bluegiga.BLE.Events.Attributes;
+using Microsoft.Extensions.Logging;
+using SharpBrick.PoweredUp.Bluetooth;
+
+namespace SharpBrick.PoweredUp.BlueGigaBLE
+{
+    public class BlueGigaBLEPoweredUpBluetoothCharacteristic : IPoweredUpBluetoothCharacteristic, IDisposable, IBlueGigaLogger
+    {
+        #region Properties
+        /// <summary>
+        /// The Service this Characteristic belongs to; will be set in the calling/constructing Service
+        /// </summary>
+        public BlueGigaBLEPoweredUpBluetoothService myService { get; set; }
```

Comment:
This "my" prefix is also very uncommon in .NET

Reply:
also taken away these unneeded prefixes :-)